src/main/java/Duke.java

```diff
@@ -31,64 +30,70 @@ else if(line.contains("done")) {
                 markAsDone(line, tasks, taskAmount);
             }
             else {
-                tasks[taskAmount] = new List(line);
+                if(line.contains("todo")) {
+                    tasks[taskAmount] = new Todo(line);
+                }
+                else if(line.contains("deadline")) {
+                    tasks[taskAmount] = new Deadline(line);
+                }
+                else if(line.contains("event")) {
+                    tasks[taskAmount] = new Event(line);
+                }
+                else {
+                    tasks[taskAmount] = new Task(line);
+                }
+                tasks[taskAmount].doneAddTask();
```

Comment:
Great implementation! I think you could keep indentation within the previous level.